### PR TITLE
Missing spaces, batch 11

### LINKED ISSUE #18019: Missing whitespace space between text and {{optional_inline}} making it one word
### MDN URL

https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/reload

### What specific section or headline is this issue about?

Parameters

### What information was incorrect, unhelpful, or incomplete?

`code` tags followed by `{{optional_inline}}` is not separated by white-space.

A couple of issues besides a bit overly tight rendering:

* Double-clicking `code` tags where one have optional-inline selects both the code *and* the *"Optional"*.

image => image

* In text-readers it is also not as readable:

image

---

The markdown and hence HTML have no white-space between `parameters` and {{optional_inline}}. For example:

```
- `tabId`{{optional_inline}}
```

Resulting in HTML:

```html
<dt id="tabid"><code>tabId</code><span class="badge inline optional">Optional</span></dt>
```

A single space between `code` and `{{optional_inline}}` should suffice. As it is now the browser interprets it as *one word*. 

Not sure how wide-spread this is, but have seen it on the one I checked. Should be possible to fix this with some script instead of manually editing each page?

### What did you expect to see?

Words are separated with space, even if they are in separate HTML inline tags.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/mozilla/add-ons/webextensions/api/tabs/reload`
* MDN URL: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/tabs/reload
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/mozilla/add-ons/webextensions/api/tabs/reload/index.md
* Last commit: https://github.com/mdn/content/commit/e49e7cbda610a687423ab4ab350c46c6f6213d6d
* Document last modified: 2022-05-07T03:05:48.000Z

</details>

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Fixes https://github.com/mdn/content/issues/18019.
Continuing PR #18123.

### CODE REVIEW

File: files/en-us/learn/forms/how_to_build_custom_form_controls/index.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ The required styles are those necessary to handle the three states of our contro
 ```css
 .select {
   /* This will create a positioning context for the list of options;
-     adding this to .select{{cssxref(':focus-within')}} will be a better option when fully supported
+     adding this to .select {{cssxref(':focus-within')}} will be a better option when fully supported
```
Comment (teoli2003) [inline files/en-us/learn/forms/how_to_build_custom_form_controls/index.md:130]: This one is correct, we don't let a space between a class selector and its pseudo-class.
```suggestion
     adding this to .select{{cssxref(':focus-within')}} will be a better option when fully supported
```

**Comment by OnkarRuikar** [comment]
@estelle, @teoli2003 what to do with these cases?

1. `{{securecontext_header}}{{APIRef("WebHID API")}}{{SeeCompatTable}}`
    Do we add spaces for these macros as well?  

2. `element.{{domxref("element.innerHTML", "innerHTML", "", "1")}}`
    Do we change it to `{{domxref("element.innerHTML", "element.innerHTML", "", "1")}}`?

3. `non-{{glossary("Cache", "cacheable")}}`
    Should we keep these as it is?

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Only one (a pseudo-class) was legit, the others were correctly fixed.

**Review by teoli2003 — APPROVED** [review]

**Comment by OnkarRuikar** [comment]
Should we add spaces to `{{securecontext_header}}{{APIRef("WebHID API")}}{{SeeCompatTable}}` these as well?

**Comment by teoli2003** [comment]
> Should we add spaces to `{{securecontext_header}}{{APIRef("WebHID API")}}{{SeeCompatTable}}` these as well?

No, `SecureContext` is a banner, as well as `SeeCompatTable` and the `APIRef` generate the sidebar. No need for spaces there.

**Pull request merged**